src/m365/spe/commands/container/container-list.ts

```diff
@@ -0,0 +1,113 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { formatting } from '../../../../utils/formatting.js';
+import { odata } from '../../../../utils/odata.js';
+import { validation } from '../../../../utils/validation.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+import { ContainerProperties } from '../../ContainerProperties.js';
+import { ContainerTypeProperties, spo } from '../../../../utils/spo.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  containerTypeId?: string;
+  containerTypeName?: string;
+}
+
+class SpeContainerListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.CONTAINER_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all Container Types';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'containerTypeId', 'createdDateTime'];
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
```

Comment:
please also include `this.#initTypes();` mapping both TypeId and TypeName to string